modules/exploits/linux/http/vestacp_exec.rb

```diff
@@ -264,12 +267,12 @@ def start_http_server
       ssl_restore = true
       datastore['SSL'] = false
     end
-    start_service({'Uri' => {
-        'Proc' => Proc.new { |cli, req|
-          on_request_uri(cli, req)
-        },
-        'Path' => resource_uri
-    }})
+    start_service({ 'Uri' => {
+      'Proc' => proc do |cli, req|
```

Comment:
Originally wasn't so sure about this, but after testing this module again against its target it seems to be fine and the documentation at https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Proc explains this change well.